src/main/java/com/openshift/jenkins/plugins/pipeline/Deployment.java

```diff
@@ -160,6 +168,8 @@ public static boolean didAllImagesChangeIfNeeded(String buildConfig, TaskListene
 		return true;
 	}
 
+	/* TODO: This appears to be dead code?
```

Comment:
Yep, it is not called.  I must have replaced it with didAllImagesChangeIfNeeded and forgot to remove it.